### PR TITLE
readme.md

### PR DESCRIPTION
**Pull request by Abhijrathod**
updated

### CODE REVIEW

File: .github/workflows/rust.yml
Comment (tronical) [inline .github/workflows/rust.yml:1]: I suggest to skip the commit that adds this file - I believe this is well covered by ci.yaml.

File: README.md [hunk 1]
```diff
@@ -40,10 +40,10 @@ follows:
 - **Embedded**: *Ready*. Slint is being used by customers in production on embedded
     devices running embedded Linux and Windows. The Slint run-time requires less than
     300KiB of RAM and can run on different processor architectures such as ARM Cortex M,
-    ESP32, STM32 from the MCU category to ARM Cortex A, Intel x86 from the MPU category.
+    ESP32, STM32 from the MCU category to ARM Cortex-A, Intel x86 from the MPU category.
 - **Desktop**: *In Progress*. While Slint is a good fit on Windows, Linux and Mac,
     we are working on improving the platform support in subsequent releases.
-- **Web**: *In Progress*. Slint apps can be compiled to WebAssembly and can run
+- **Web**: *In Progress*. Slint apps can be compiled into WebAssembly and can run
```
Comment (tronical) [inline README.md:46]: I think `compile to` is idiomatic, I'd prefer to stay with `to` over `into`.

File: README.md [hunk 2]
```diff
@@ -82,14 +82,14 @@ support for users with special needs.
 
 ### Hello World
 
-The UI is defined in a Domain Specific Language that is declarative, easy to use,
+The UIkeyboard-based a Domain Specific Language that is declarative, easy to use,
```
Comment (tronical) [inline README.md:85]: This looks like an unintentional change?

File: README.md [hunk 3]
```diff
@@ -82,14 +82,14 @@ support for users with special needs.
...
 intuitive, and provides a powerful way to describe graphical elements, their
 placement, their hierarchy, property bindings, and the flow of data through the
 different states.
 
 Here's the obligatory "Hello World":
 
-```slint
+```saint
```
Comment (tronical) [inline README.md:92]: This also looks like a typo?

File: README.md [hunk 4]
```diff
@@ -115,7 +115,7 @@ The `docs` folder contains a lot more information, including
 [build instructions](docs/building.md), and
 [internal developer docs](docs/development.md).
 
-Refer to the README of each language directory in the `api` folder:
+Refer to the README of each language directory in the `API` folder:
```
Comment (tronical) [inline README.md:118]: I think the lower-case spelling is correct as here `api` refers to a folder name, not the abbreviation.

File: README.md [hunk 5]
```diff
@@ -136,8 +136,8 @@ are pure functions that the compiler can optimize. For example, the compiler
 could choose to "inline" properties and remove those that are constant or
 unchanged. In the future we hope to improve rendering time on low end devices by
 pre-processing images and text. The compiler could determine that a `Text` or an
-`Image` element is always on top of another `Image` in the same location.
-Consequently both elements could be rendered ahead of time into a single
+The `Image` element is always on top of another `Image` in the same location.
```
Comment (tronical) [inline README.md:139]: That looks wrong to me, the "`Image` element ..." part is part of an entire sentence.

File: README.md [hunk 6]
```diff
@@ -159,13 +159,13 @@ Rendering backends and styles are configurable at compile time:
 - The `software` renderer uses the CPU with no additional dependencies.
 
 NOTE: When Qt is installed on the system, the `qt` style becomes available,
-using Qt's QStyle to achieve native looking widgets.
+using Qt's QStyle to achieve nativnative-lookingets.
```
Comment (tronical) [inline README.md:162]: This looks like a typo?

File: README.md [hunk 7]
```diff
@@ -159,13 +159,13 @@ Rendering backends and styles are configurable at compile time:
...
 
 ### Tooling
 
 We have a few tools to help with the development of .slint files:
 
-- A [**LSP Server**](./tools/lsp) that adds features like auto-complete and live
+- A [**LSP Server**](./tools/lsp) LSPt adds features like auto-complete and live
```
Comment (tronical) [inline README.md:168]: This also looks like a mistake?

File: README.md [hunk 8]
```diff
@@ -206,9 +206,8 @@ Please see our separate [FAQ](FAQ.md).
 ## About us (SixtyFPS GmbH)
 
 We are passionate about software - API design, cross-platform software
-development and user interface components. Our aim is to make developing user
-interfaces fun for everyone: from JavaScript, C++, or Rust developers all the
-way to UI/UX designers. We believe that software grows organically and keeping
+development, and user interface components. We aim to make developing user
+interfaces fun for everyone: from JavaScript, C++, or Rust developers to UI/UX designers. We believe that software grows organically and keeping
```
Comment (tronical) [inline README.md:210]: Why did you remove the "all the way" part?

File: README.md [hunk 9]
```diff
@@ -224,7 +223,7 @@ Feel free to join [Github discussions](https://github.com/slint-ui/slint/discuss
 for general chat or questions. Use [Github issues](https://github.com/slint-ui/slint/issues)
 to report public suggestions or bugs.
 
-We chat in [our Mattermost instance](https://chat.slint.dev) where you are
+We chat in [our Mattermost instance](https://chat.slint.dev) Welcome  are
```
Comment (tronical) [inline README.md:226]: What is the reason for this change?

**Review by tronical — CHANGES_REQUESTED** [review]
Thank you for your contribution. Many of the suggested changes to the README look like a concrete improvement. There are also changes in there that I don't quite understand or that look like typos to me.

**Comment by ogoffart** [comment]
Thanks for the pull request, but i'm going to close this.
Half of the changes are bogus.

**Pull request closed without merging**